### ISSUE TITLE
black out drunk personality commune button does nothing

### ISSUE THREAD

**Issue by kawoppi** [tgstation/tgstation#79243]
## Round ID:
[217428](https://scrubby.melonmesa.com/round/217428)

## Testmerges:
The following pull requests are currently test merged:
[#79236: 'Reworks the styling of the announcements and expands major announcements to also be colourable.' by Watermelon914 at commit aa627b45c9](https://www.github.com/tgstation/tgstation/pull/79236)

## Reproduction:
Keep drinking until you get blackout drunk and somebody else takes over. Press the Personality Commune button to try and talk to them. No chat window pops up, nothing seems to happen.

**Comment by Treachce** [comment]
That might be intentional since the whole point of being blackout drunk is that you completely lose control, being able to tell whoever's possessing you what to do when drunk would defeat the purpose.

From personality_commune.dm
``` DM
	var/datum/brain_trauma/severe/split_personality/trauma = target
	if(!istype(trauma)) // hypothetically impossible but you never know
		return . | SPELL_CANCEL_CAST
```
The communion gets cancelled if the split personality isn't the trauma subtype,

split_personality.dm
```DM
/datum/brain_trauma/severe/split_personality/proc/make_backseats()
	stranger_backseat = new(owner, src)
	var/datum/action/cooldown/spell/personality_commune/stranger_spell = new(src)
	stranger_spell.Grant(stranger_backseat)

	owner_backseat = new(owner, src)
	var/datum/action/cooldown/spell/personality_commune/owner_spell = new(src)
	owner_spell.Grant(owner_backseat)
```
but the spells seem to be granted for all split_personality types anyways.

**Comment by wesoda25** [comment]
fixed as per #79631

**Comment by kawoppi** [comment]
wahoo!
